.travis.yml

```diff
@@ -9,5 +9,9 @@ rvm:
 
 sudo: false
 
+before_install:
+  - gem uninstall -v '>= 2' -i $(rvm gemdir)@global -ax bundler || true
+  - gem install bundler -v '< 2'
```

Comment:
I'm missing some context here. Why not specify this in the gemspec?

Reply:
You need bundler installed to understand gemspecs (bit of a 🐔and 🥚problem)... Also I think you can only specify the version from within a `Gemfile` (which this project doesn't have)

Reply:
👍